include/libcyphal/transport/can/transport.hpp

```diff
@@ -654,14 +644,22 @@ class TransportImpl final : public ICanTransport, private TransportDelegate  //
 
 }  // namespace detail
 
-CETL_NODISCARD inline Expected<UniquePtr<ICanTransport>, FactoryError> makeTransport(
-    cetl::pmr::memory_resource&  memory,
-    IMultiplexer&                multiplexer,
-    const cetl::span<IMedia*>    media,
-    const std::size_t            tx_capacity,
-    const cetl::optional<NodeId> local_node_id)
+/// @brief Makes a new CAN transport instance.
+///
+/// NB! Lifetime of the transport instance must never outlive `memory` and `media` instances.
+///
+/// @param memory Reference to a polymorphic memory resource to use for all allocations.
+/// @param media Collection of redundant media interfaces to use.
+/// @param tx_capacity Total number of frames that can be queued for transmission.
+/// @param local_node_id Optional id of the local node. Could be set (once!) later by `setLocalNodeId` call.
```

Comment:
```suggestion
/// @param local_node_id Optional node-ID of the local node. Could be set (once!) later by `setLocalNodeId` call.
```

It is best to use the terms from the Specification verbatim.

Although, it makes me wonder if we should remove this now that we have `setLocalNodeId()`?

Reply:
I think it is okey - you might have your node id right from the beginning (from registry or whatever persisting storage).

Reply:
But in that case you can just call setLocalNodeId later

Reply:
ok, it's done